### DISCUSSION TITLE
Renovate bot not respecting branch creation schedule - creates branches outside schedule

### DISCUSSION THREAD

**Discussion by rustyb** [renovatebot/renovate#18438] [Request Help]
Hi there - I'm trying to get to the bottom of renovate not respecting the schedule defined in our config.

As you can see in the screenshot below renovate is reports the configuration we have in our renovate.json correctly:

```json
{
  "automergeSchedule": ["after 10am and before 4pm every weekday"],
  "schedule": ["after 7am every weekday", "before 5pm every weekday"],
  "timezone": "Europe/Amsterdam"
}
```

But it just created a branch and PR at 20h51 which is outside the schedule. 
![Screenshot 2022-10-20 at 20 52 42](https://user-images.githubusercontent.com/53086/197033977-4a5d054d-85de-4dbd-b55c-e387c7af7137.png)

I'm pretty sure a month ago it was respecting the schedule.

Anyone have any idea how to stop this from happening?

**Comment by viceice** [comment]
you need to combine schedule, as the array is combined by `or`. so you need to use `and` like in your automerge schedule.

**Reply by rustyb** [reply]
Doh that's my bad thanks 👍
